lib/hammer_cli_katello/id_resolver.rb

```diff
@@ -72,6 +73,16 @@ def content_view_version_id(options)
         # Intentionally suppressing the exception,
         # environment is not always required.
       end
+
+      begin
+        options[key_content_view_id] ||= content_view_id(
+          scoped_options("content_view", options)
+        )
+      rescue HammerCLIForeman::MissingSeachOptions # rubocop:disable HandleExceptions
+        # Intentionally suppressing the exception,
+        # content_view is not always required.
+      end
```

Comment:
Without changing HammerCLIForeman's id_resolver, i don't believe so.  This is used in other places.